apps/app.near/widget/sdk/social.jsx

```diff
@@ -0,0 +1,27 @@
+const SocialSDK = {
+  poke: (accountId) => {
+    console.log(accountId);
+    const data = {
+      index: {
+        graph: JSON.stringify({
+          key: "poke",
+          value: {
+            accountId: accountId,
+          },
+        }),
+        notify: JSON.stringify({
+          key: accountId,
+          value: {
+            type: "poke",
+          },
+        }),
+      },
+    };
+
+    Social.set(data, {
+      force: true,
+    });
+  },
+};
+
```

Comment:
Could we move here follow, post, and comment, too

Reply:
What do you mean move post and comment to sdk?